.github/workflows/integration_test.yml

```diff
@@ -34,13 +39,18 @@ jobs:
           sleep 60
           kubectl logs $(kubectl -n kepler get pods -oname) -n kepler
           kubectl get all -n kepler
-
+      
       - name: run integration_test
         run: |
           docker ps -a
           mkdir -p /tmp/.kube
           kind get kubeconfig --name=kind > /tmp/.kube/config
-          kubectl port-forward $(kubectl -n kepler get pods -oname) 9102:9102 -n kepler -v7 &
+          docker network ls
```

Comment:
can you check cni network too?

Reply:
I don't have the approach to hack into kind and check cni for now.
so far, it seems that kind network defined in docker is not same with the cni from the uuid.

Reply:
I suppose some thing we can do is that to upgrade kind and kubectl to see any changes happen?

Reply:
related with https://github.com/moby/moby/issues/17666